examples/credential_metadata_jwt_vc_json.json

```diff
@@ -13,35 +13,40 @@
         "type": [
           "VerifiableCredential",
           "UniversityDegreeCredential"
-        ],
-        "credentialSubject": {
-          "given_name": {
-            "display": [
-              {
-                "name": "Given Name",
-                "locale": "en-US"
-              }
-            ]
-          },
-          "family_name": {
-            "display": [
-              {
-                "name": "Surname",
-                "locale": "en-US"
-              }
-            ]
-          },
-          "degree": {},
-          "gpa": {
-            "mandatory": true,
-            "display": [
-              {
-                "name": "GPA"
-              }
-            ]
-          }
-        }
+        ]
       },
+      "claims": [
+        {
+          "path": ["credentialSubject", "given_name"],
+          "display": [
+            {
+              "name": "Given Name",
+              "locale": "en-US"
+            }
+          ]
+        },
+        {
+          "path": ["credentialSubject", "family_name"],
+          "display": [
+            {
+              "name": "Surname",
+              "locale": "en-US"
+            }
+          ]
+        },
+        {
+          "path": ["credentialSubject", "degree"]
```

Comment:
Is it intentional that there is no "display" element for the degree?

Reply:
#432